float64 excess_heat
---
bool success
string message
